source/playbooks/SC/ssmdocs/SC_Config.1.ts

```diff
@@ -34,7 +34,7 @@ export class EnableAWSConfigDocument extends ControlRunbookDocument {
   protected getRemediationParams(): { [_: string]: any } {
     const params = super.getRemediationParams();
 
-    params.SNSTopicName = `${this.solutionId}-${this.solutionAcronym}-AWSConfigNotification`;
+    params.SNSTopicName = `${this.solutionId}-SHARR-AWSConfigNotification`;
```

Comment:
is this one of the resources that didn't get updated when we ran the SHARR->ASR name conversion?

Reply:
We fixed those in the 1.5.1 branch - this same control runbook exists in the other standards/playbooks. When we merged back into develop, these (SC/CIS 1.4) were already there and I didn't think to fix them at the time.

Reply:
so is this something that can be fixed now, or would doing so cause additional breakages that we don't want to include in this release?

Reply:
What do you mean by fix? Change it to ASR? Correct, we have not assessed the impact of doing that.

Reply:
yes, change to ASR. If the impact isn't already covered by what we did before then yeah this prob isn't worth doing